tests/schemas/test_pydantic.py

```diff
@@ -64,7 +64,7 @@ class Meta:
 
 
 def test_pydantic_record_schema_logical_types(logical_types_schema):
-    a_datetime = datetime.datetime(2019, 10, 12, 17, 57, 42)
+    a_datetime = datetime.datetime.fromtimestamp(1570903062, datetime.timezone.utc)
```

Comment:
think it is better to use something more human readable, otherwise we will have to execute the code to see what is generating and there is not too much difference with the previous approach